packages/drivers/routerlicious-driver/src/wholeSummaryDocumentStorageService.ts

```diff
@@ -323,16 +324,13 @@ export class WholeSummaryDocumentStorageService implements IDocumentStorageServi
 		normalizedWholeSummary: INormalizedWholeSummary,
 		versionId: string | null,
 	): Promise<string> {
-		const wholeFlatSummaryId = normalizedWholeSummary.snapshotTree.id;
-		assert(wholeFlatSummaryId !== undefined, 0x275 /* "Root tree should contain the id" */);
+		const snapshotId = normalizedWholeSummary.id;
```

Comment:
can you add a bit more data on what this fixes / changes? I.e. scenario.

Reply:
done

Reply:
Added in description.

Reply:
Thanks! So I think you are saying that (from user POV) we have a regression after previous PR and this PR fixes it. Might be worth capture it more clearly, as some of these PRs will be ported to main, so any instructions you give in PRs (including last one that caused regression) would be super useful

Reply:
The link to PR is in description. Although it was not a regression as I was just caching with different id and was supplying that id to loader also. So it` would load from cache only. But this is the correct way.